src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue

```diff
@@ -0,0 +1,106 @@
+<template>
```

Comment:
Not sure if this deserves a new component, its very much like `KivaClassicMultiCategoryCarousel.vue` but only 1 category and the description and name of the category added. I tried using `KivaClassicMultiCategoryCarousel.vue` with a single category and having the description and name in the parent component, but the ideal UI is for the description and name to not show up until the carousel starts loading, which is why I went with a new component -- alternatively we could add some more props to the multi carousel to optionally display the description and name

Reply:
I'm glad you could at least reuse the carousel component. It has one major flaw right now which is that it needs a min height that matches the typical height of the loan card so that we don't have a giant layout shift when the content does finally load. Ideally we'd have a placeholder state for the name and description along with the loan cards.

In what scenario would a single row carousel not be loading? Lazy-loading down the page?

Reply:
The page would load all the titles and the descriptions, which looks weird, and then there would be at least a 1sec delay before the loading state carousels showed up. This is due to the fetchLoans call resolving